probe-rs/src/bin/probe-rs/cmd/dap_server/server/configuration.rs

```diff
@@ -18,8 +18,7 @@ pub struct SessionConfig {
     pub(crate) cwd: Option<PathBuf>,
 
     /// The debug probe selector associated with the debug probe to use. Use 'list' command to see available probes
-    #[serde(alias = "probe")]
-    pub(crate) probe_selector: Option<DebugProbeSelector>,
+    pub(crate) probe: Option<DebugProbeSelector>,
```

Comment:
@noppej am I breaking something important by removing the longer name?

Reply:
@bugadani , You are good. VSCode still uses the "probe" option.